themes/_default.yaml

```diff
@@ -513,11 +513,12 @@ sliding-panel:
       font-size: "32px"
       height: "25px"
       width: "20px"
-      right: "15px"
     back-block:
       font-size: "13px"
       icon:
         font-size: "25px"
+    block:
+      lateral-margin: "15px"
```

Comment:
Is this value different across the themes?

Reply:
This value (as I remember) I took from `right` variable. The cross was postponed in 15px, and from left side can be `back` arrow. And when `back` arrow is absent, we have wrong aligning, that's why I just move this value to common variable and apply it to both sides.